### PR TITLE
feature CSI-3190 rename bundles

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

**Review by zingero — APPROVED** [review]

**Review by zingero — APPROVED** [review]

**Review by oriyarde — DISMISSED** [review]
LGTM

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -119,7 +119,7 @@ metadata:
         }
       ]
 spec:
-  displayName: "Operator for IBM block storage CSI driver"
+  displayName: "IBM block storage CSI driver plugin"
```
Comment (erantzabari) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml:122]: Should be "IBM block storage CSI driver operator"

File: deploy/olm-catalog/ibm-block-csi-operator/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -111,7 +111,7 @@ metadata:
         }
       ]
 spec:
-  displayName: "Operator for IBM block storage CSI driver"
+  displayName: "IBM block storage CSI driver plugin"
```
Comment (erantzabari) [inline deploy/olm-catalog/ibm-block-csi-operator/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml:114]: hould be "IBM block storage CSI driver operator"

**Review by erantzabari — CHANGES_REQUESTED** [review]

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml (hunk 1 above)
Comment (ArbelNathan) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml]: done

File: deploy/olm-catalog/ibm-block-csi-operator/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml (hunk 2 above)
Comment (ArbelNathan) [reply deploy/olm-catalog/ibm-block-csi-operator/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml]: done

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by erantzabari — APPROVED** [review]

**Pull request merged**